tests/avro/read_test_schema.avsc

```diff
@@ -0,0 +1,7 @@
+{
+ "type": "record",
+ "name": "UserKey",
```

Comment:
I believe the two schema must have the same record name in order to satisfy the schema resolution rules laid out in the Avro documentation. 

http://avro.apache.org/docs/current/spec.html#Schema+Resolution

```
It is an error if the two schemas do not match.
...
both schemas are records with the same name
```

Reply:
Thanks, makes sense. Naming it `UserKey` was anyway not optimal. However, I fixed it by means of another Avro schema evolution feature: aliases (e902f2f)